app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,63 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.models.Task;
+import com.codesoom.assignment.services.TaskService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+@RestController
+@RequestMapping("/tasks")
+public class TaskController {
+
+    private final TaskService taskService;
+
+    public TaskController(TaskService taskService) {
+        this.taskService = taskService;
+    }
+
+    @GetMapping("")
+    public ResponseEntity<?> getAllTask() {
+        return new ResponseEntity<>(taskService.readTasks(), HttpStatus.OK);
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<?> getTask(@PathVariable Long id) {
+        if(taskService.ReadTaskIsNull(id)) {
+            return new ResponseEntity<>("", HttpStatus.NOT_FOUND);
+        }
+
+        return new ResponseEntity<>(taskService.readTask(id), HttpStatus.OK);
+    }
+
+    @PostMapping
+    public ResponseEntity<?> createTask(@RequestBody Task task) {
+        return new ResponseEntity<>(taskService.createTask(task), HttpStatus.CREATED);
+    }
```

Comment:
`ResponseEntity`를 이용해서 응답 상태코드를 변경해주신 것 같아요.

`@ResponseStatus(HttpStatus.CREATED)`데코레이터로 상태코드를 변경하고, 응답은 Task를 그대로 돌려주면
더 간단하게 문제를 해결할 수 있을 것 같아요

```suggestion
    @PostMapping
    @RequestMapping(HttpStatus.CREATED)
    public Task createTask(@RequestBody Task task) {
        return taskService.createTask(task);
    }
```

Reply:
코멘트에 @RequestMapping 오타가 났네요 ㅎㅎ 
이렇게 보내는 기능이 있는지 몰랐습니다. 적용해봤습니다.
그리고 ResponseEntity를 사람들이 많이 사용하던데 음.. 일반 Task로 반환하시도록 하는 이유가 있을까요?

Reply:
맞네요 ㅎㅎ `@ResponseStatus`가 맞습니다 ㅎㅎ

저는 `ResponseEntity`를 사용할 필요가 없어서 일반 Task로 반환하는게 좋은 것 같습니다